config/packages/dev/messenger.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+    messenger:
+        # Uncomment this (and the failed transport below) to send failed messages to this transport for later handling.
+        # failure_transport: failed
+
+        transports:
+            async: 'in-memory://'
```

Comment:
Is hierdoor niet alles async? Is dat wat we verwachten?

Reply:
Puur voor te testen. Enkel de code uit src/ zal overgenomen worden naar echte projecten. We testen onze handlers door ze direct te callen als classes (zie https://github.com/sumocoders/Framework-User-Implementation-Example/blob/main/tests/UserTest.php#L35), zoals vroeger in Fork. Messages worden niet effectief verzonden dus er hoeft geen echt transport te zijn.